rom/dev/src/flow/warm_reset.rs

```diff
@@ -25,11 +25,12 @@ impl WarmResetFlow {
     ///
     /// * `env` - ROM Environment
     #[inline(never)]
-    pub fn run(env: &mut RomEnv) -> CaliptraResult<FirmwareHandoffTable> {
+    pub fn run(_env: &mut RomEnv) -> CaliptraResult<FirmwareHandoffTable> {
         cprintln!("[warm-reset] ++");
 
         cprintln!("[warm-reset] --");
 
-        Ok(fht::make_fht(env))
+        // Caller does not expect an FHT in the case of a warm reset.
```

Comment:
If this statement is true, perhaps this function should return CaliptraResult<Option<FirmwareHandoffTable>> rather than a bogus FHT?